app/queries/update_user_mutator.rb

```diff
@@ -0,0 +1,63 @@
+class UpdateUserMutator < ApplicationQuery
+  property :id, validates: { presence: true }
+  property :name, validates: { presence: true }
+  property :about, validates: { length: { maximum: 1000 } }
+  property :current_password
+  property :new_password
+  property :confirm_new_password
```

Comment:
All text input should have limits. These should probably have `validates: { length: { minimum: 8, maximum: 128 }, allow_blank: true }` - this matches the validation for the _external_ password reset form.

This also means the `passwords_should_be_secure` validation method isn't required.